src/javascript/Root.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component, PropTypes } from 'react'
+import { Redirect, Router, Route } from 'react-router'
+import { Provider } from 'react-redux'
+import { store } from './redux'
+import mui from 'material-ui'
+import theme from './config/theme'
+
+const ThemeManager = new mui.Styles.ThemeManager();
+ThemeManager.setTheme(theme);
+
+// Views
```

Comment:
can you get rid off some of the comments, I'm not huge fan of them. Think what we can move to the docs, leave them unless they are really necessary. Thanks